civihr_employee_portal/js/reports.js

```diff
@@ -393,12 +407,14 @@
    * Init the scrollbar fallback
    *
    */
-  HRReport.prototype.initScrollbarFallback = function () {
+  HRReport.prototype.initScrollbarFallbackOnTabChange = function () {
     var el = document.querySelector('.chr_custom-scrollbar');
 
-    if (el) {
-      Ps.initialize(el);
-    }
+    $('[data-tab="view-data"]').click(function () {
+      setTimeout(function () {
```

Comment:
Why do we need a timeout?

Reply:
`timeout is added to wait for the tabs component to properly display the data:`
can we use `shown.bs.tab` event to wait for the tab to be shown properly, and eliminate timeout?

Reply:
These tabs are not using Bootstrap's tabs, but [custom code](https://github.com/compucorp/civihr-employee-portal/blob/staging/civihr_employee_portal/js/reports.js#L680). Simply changing the tabs to Bootstrap doesn't work since the URL switch functionality is lost and the tab switch events are not triggered.

For now I have moved the scroll initialization to the Behaviour, in this way we avoid the setTimeout.